lib/api/model/initial_snapshot.dart

```diff
@@ -86,6 +90,10 @@ class InitialSnapshot {
     required this.zulipMergeBase,
     required this.alertWords,
     required this.customProfileFields,
+    // TODO(server-8): Remove the default values.
+    this.serverTypingStartedExpiryPeriodMilliseconds = 15000,
```

Comment:
Good thought.

Let's use `JsonKey.defaultValue` to express this instead, though:
https://pub.dev/documentation/json_annotation/latest/json_annotation/JsonKey/defaultValue.html

That way these fallbacks are a bit more clearly demarcated as part of the processing of the JSON data, rather than being part of the constructor for the class.

Reply:
This originally used `defaultValue`, but that would add repetition to the example data. In that sense we might want to define the fallbacks as `static const`s for re-use purposes.

Reply:
I think it's fine for the example data to say the value again — in fact it's preferable for it to do so.

One angle on that: after all, once we discharge that `TODO(server-8)` comment, the example data will need to specific the value for itself because there won't be a fallback value in the live code.

Abstracting a bit: although it's not entirely a coincidence that the value we fall back to for old servers, and the boring value we want to use in tests where this property isn't intended to be interesting, are equal… it didn't have to be the case that they're equal, and they are distinct concepts. So the example data should say the value it wants as the boring example value, while the deserialization in the live code says the data it wants to fall back to for old servers.